packages/api/src/external/carequality/document/query-documents.ts

```diff
@@ -40,7 +42,11 @@ export async function getDocumentsFromCQ({
     ]);
 
     if (!cqPatientData || cqPatientData.data.links.length <= 0) {
-      return interrupt(`Patient has no CQ links, skipping DQ`);
+      // If DQ is triggered while the PD is in progress, schedule it to be done when PD is completed
+      if (getCQData(patient.data.externalData)?.discoveryStatus === "processing") {
```

Comment:
shouldn't this be outside `if (!cqPatientData || cqPatientData.data.links.length <= 0) {`?

AFAIK PD processing should be the only condition we need to consider to schedule a DQ